packages/repo-tools/src/commands/index.ts

```diff
@@ -24,6 +24,19 @@ export function registerCommands(program: Command) {
     .option('--ci', 'CI run checks that there is no changes on API reports')
     .option('--tsc', 'executes the tsc compilation before extracting the APIs')
     .option('--docs', 'generates the api documentation')
+    .option(
+      '--allow-warnings [allowWarningsPaths...]',
+      'continue processing packages after getting errors on selected packages',
+      false,
+    )
+    .option('--folders <folders...>', 'packages folder containers', [
+      'packages',
+      'plugins',
+    ])
```

Comment:
I'm thinking we should be reading this from the root `package.json` workspace field instead?